app/experimenter/kinto/tests/test_serializers.py

```diff
@@ -0,0 +1,83 @@
+import os
+import json
+from datetime import datetime
+from jsonschema import validate
+from django.test import TestCase
+
+from experimenter.experiments.models import Experiment
+from experimenter.experiments.tests.factories import (
+    ExperimentFactory,
+    ExperimentVariantFactory,
+)
+from experimenter.kinto.serializers import ExperimentRapidRecipeSerializer
+
+
+class TestExperimentRapidSerializer(TestCase):
+    def test_serializer_outputs_expected_schema(self):
+        audience = Experiment.RAPID_AUDIENCE_CHOICES[0][1]
+        features = [feature[0] for feature in Experiment.RAPID_FEATURE_CHOICES]
+        normandy_slug = "experimenter-normandy-slug"
+        experiment = ExperimentFactory.create(
+            type=Experiment.TYPE_RAPID,
+            rapid_type=Experiment.RAPID_AA_CFR,
+            audience=audience,
+            features=features,
+            normandy_slug=normandy_slug,
+            proposed_enrollment=9,
+        )
+
+        ExperimentVariantFactory.create(
+            experiment=experiment, slug="control", is_control=True
+        )
+        ExperimentVariantFactory.create(experiment=experiment, slug="variant-2")
+
+        serializer = ExperimentRapidRecipeSerializer(experiment)
+        data = serializer.data
+
+        fn = os.path.join(os.path.dirname(__file__), "experimentRecipe.json")
```

Comment:
What does `fn` mean?  I usually call this kind of thing a 'path'.  Also it completely doesn't matter since we're getting rid of this in a week anyways, just curious.

Reply:
lol originally i had it as fn for "file name" but then realized i needed the path